shared/lib_irradproc.cpp

```diff
@@ -903,6 +942,11 @@ calculate_spa(double jd, double lat, double lng, double alt, double pressure, do
         azimuth = M_PI;
     }
 
+    std::vector<double> spa_outputs = {needed_values[0], needed_values[1], needed_values[2], needed_values[3], needed_values[4], needed_values[5], needed_values[6], needed_values[7], needed_values[8],
+        ascension_and_declination[0], ascension_and_declination[1]};
+
+    if (use_table)
+        spa_table[spa_key_inputs] = spa_outputs;
```

Comment:
Have you run with use_table = false to verify previous results?

Reply:
Yes. I'm planning to remove this code once we're done examining sensitivities-- thoughts on deleting this?